docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -103,30 +59,30 @@ If you're using Jira Cloud, navigate to [Account Settings → Security → API T
 
 For Jira Server, the URL of the API tokens page will be different depending on your installation.
 
-### Setting up Jira Webhooks
+### Setting up Jira webhooks
 
-Go to Settings → General → System → Webhooks and create a new Webhook for Jira to tell the Teleport Plugin about updates.
+Go to Settings → General → System → Webhooks and create a new webhook for Jira to tell the Teleport plugin about updates.
 
-For the webhook URL, use the URL that you'll run the plugin on. It needs to be a publicly accessible URL that we'll set up later. Jira requires the webhook listener to run over HTTPS.
+For the webhook URL, use the URL that you'll run the plugin on. It needs to be a publicly accessible URL (we will show you how to set this up later). Jira requires the webhook listener to run over HTTPS.
 
-The Teleport Jira plugin webhook needs to be notified only about new issues being created, issues being updated, or deleted. You can leave all the other boxes empty.
+The webhook needs to be notified only about new issues being created, issues being updated, or deleted. You can leave all the other boxes empty.
 
 <Admonition
   type="note"
   title="Plugin Defaults"
 >
-  Jira Webhook will send updates about any issues in any projects in your Jira installation to the webhook.  We suggest that you use JQL filters to limit which issues are being sent to the plugin.
+  Jira will send updates about any issues in any projects in your Jira installation to the webhook. We suggest that you use JQL filters to limit which issues are being sent to the plugin.
 
-  The Plugin's web server will run with TLS, but you can disable it with `--insecure-no-tls` to test things out in a dev environment.
+  The plugin's web server will run with TLS, but you can disable it with `--insecure-no-tls` to test things out in a dev environment.
 </Admonition>
 
 In the webhook settings page, make sure that the webhook will only send Issue Updated updates. It's not critical if anything else gets sent, the plugin will just ignore everything else.
```

Comment:
There should be a semicolon (;) instead of a comma after "It's not critical if anything else gets sent".